docs/glossary.rst

```diff
@@ -478,10 +478,42 @@ defined role within the organization (see our reference on :ref:`user roles
 
     :ref:`User roles <user-roles>`
 
+.. _gloss-version:
+
+Versions
+--------
+
+CrateDB uses a versioning system with three levels of versioning: major
+versions, minor versions, and patch versions. (Versions can also be referred to
+as releases.) These apply also to CrateDB Cloud clusters.
+
+A major version of CrateDB is a release that includes significant changes in
+features, performance, and/or supported operations that are not backwards
+compatible with any previous version. It is indicated by the first numeral in
+the versioning sequence, i.e. the 4 in 'version 4.6.7'.
+
+A minor version of CrateDB is a release that includes substantial changes in
+features, performance, and/or supported operations compared to the previous
+such version. It is indicated by the second numeral in the versioning sequence,
+e.g. the 6 in 'version 4.6.7'. You can upgrade minor versions yourself in the
+CrateDB Cloud Console under
+:ref:`Cluster Preferences <overview-cluster-settings-upgrade>`.
+
+A patch version of CrateDB is a release that includes bug fixes and smaller
+quality of life improvements compared to the previous such version. It is
+indicated by the third numeral in the versioning sequence, e.g. the 7 in
+'version 4.6.7'. You can upgrade patch versions yourself in the CrateDB Cloud
+Console under :ref:`Cluster Preferences <overview-cluster-settings-upgrade>`.
+
+.. SEEALSO::
+
```

Comment:
I am wondering if we should explain that we only support  upgrading only one minor version at a time?

Reply:
I do mention that in the Console Overview section it links to - it just seemed to me that it's not really relevant for the glossary. But you can argue it either way